src/style/app.scss

```diff
@@ -169,8 +164,8 @@ mdl-simple-tooltip div {
 
 .card-selected {
   box-shadow: 0 8px 10px 1px rgba(0, 0, 0, 0.14),
-              0 3px 14px 2px rgba(0, 0, 0, 0.12),
-              0 5px 5px -3px rgba(0, 0, 0, 0.2) !important;
+  0 3px 14px 2px rgba(0, 0, 0, 0.12),
```

Comment:
@wowshakhov , please check with @andrewbents is it possible to configure editor to format the code the way it was before